lib/generators/templates/models/rental.rb

```diff
@@ -3,7 +3,7 @@ class Rental < BookingsyncPortal::Rental
     local_attributes: [:position, :published_at],
     include: [:availability],
     delegate_attributes: [:name, :sleeps, :sleeps_max, :bedrooms_count, :bathrooms_count,
-      :surface, :surface_symbol]
+      :surface, :surface_unit]
```

Comment:
@ZenCocoon ,
hm, is this kind of the hotfix? Is it possible we have issues on bsa-`channel` apps?